plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -255,23 +242,19 @@ export class HookCommander {
         })
     }
 
-    public async postRestHook(
-        hook: Hook,
-        event: PostIngestionEvent,
-        person: IngestionPersonData | undefined
-    ): Promise<void> {
+    public async postRestHook(hook: Hook, event: PostIngestionEvent): Promise<void> {
         let sendablePerson: Record<string, any> = {}
-        if (person) {
-            const { uuid, properties, team_id, id } = person
-
-            // we standardize into ISO before sending the payload
-            const createdAt = person.created_at.toISO()
+        if (event.person_id) {
+            const uuid = event.person_id
+            const properties = event.person_properties
+            const team_id = event.teamId
+            const createdAt = event.person_created_at
 
             sendablePerson = {
+                // TODO: can we drop the id from here?
                 uuid,
                 properties,
                 team_id,
-                id,
```

Comment:
Note: we're dropping the person.id from being sent with webhooks - if we don't want to drop that we'll need to do a postgres query - I suspect its better for us to drop it, users shouldn't know the PG person id